site/content/en/docs/manual/basics/attach-cloud-storage.md

```diff
@@ -8,376 +8,415 @@ description: 'Instructions on how to attach cloud storage using UI'
 
-Now you can [attach new cloud storage into CVAT](#attach-new-cloud-storage).
+1. Go to the containers section and on the top menu click **+Container**
 
-## Using Google Cloud Storage
+![](/images/azure_blob_container_tutorial7.jpg)
 
-### Create Google account
+3. Enter the name of the container.
+4. (Optional) In the **Public access level** drop-down, select type of the access.
+   <br>**Note:** this field will inactive if you disabled **Allow enabling public access on containers**.
+5. Click **Create**.
 
-First, create a Google account, go to [account login page](https://accounts.google.com/) and click `Create account`.
-After, go to the [Google Cloud page](https://cloud.google.com), click `Get started`, enter the required data
-and accept the terms of service (you'll need credit card information to register).
+### Upload data
 
-### Create a bucket
+You need to upload data for annotation and the `manifest.json` file.
 
-Your first project will be created automatically, you can see it on the [cloud resource manager page](https://console.cloud.google.com/cloud-resource-manager).
-To create a bucket, go to the [cloud storage page](https://console.cloud.google.com/storage/browser)
-and press `Create bucket`. Next, enter the name of the bucket, add labels if necessary, select the type of location
-for example region and the location nearest to you, select storage class, when selecting access control
-you can enable `Enforce public access prevention on this bucket` (if you plan to have anonymous access to your bucket,
-it should be disabled) you can select `Uniform` or `Fine-grained` access control, if you need protection of your
-object data select protect object data type. When all the information is entered click `Create` to create the bucket.
+1. Prepare data.
+   For more information,
+   see [prepare the dataset](#prepare-the-dataset).
+2. Go to container and click **Upload**.
+3. Click **Browse for files** and select images.
+   > Note: If images are in folder, specify folder in the **Advanced settings** > **Upload to folder**.
+4. Click **Upload**.
 
-![](/images/google_cloud_storage_tutorial1.jpg)
+![](/images/azure_blob_container_tutorial9.jpg)
 
-### Upload
+### SAS token
 
-Prepare the dataset as in the point [prepare dataset](#prepare-dataset).
+Use the SAS token to grant secure access to the container.
 
-To upload files, you can simply drag and drop files and folders into a browser window
-or use the `upload folder` and/or `upload files`.
+To configure the SAS token:
 
-### Access permissions
+1. Go to **Home** > **Resourse groups** > You resourse name > Your storage account.
+2. On the left menu, click **Shared access signature**.
+3. Change the following fields:
+   - **Allowed services**: Enable **Blob** . Disable all other fields.
+   - **Allowed resource types**: Enable **Container** and **Object**. Disable all other fields.
+   - **Allowed permissions**: Enable **Read** and **List**. Disable all other fields.
```

Comment:
The user should also select at least `write` permission if he will export the annotations to the container. I guess this should also be mentioned.